app/Models/Job.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Job extends Model
+{
+    use HasFactory;
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array<int, string>
+     */
+    protected $fillable = [
```

Comment:
One tip for the future, you can also just declare the model property “guarded” usually with only the table id field and that means all the rest are allowed to be filled out, at the end the security is implemented by the controllers when validating the request